scipy/stats/_probability_distribution.py

```diff
@@ -123,11 +123,19 @@ def sample(self, shape, *, method, rng):
             Not all `method` options are available for all distributions.
             If the selected `method` is not available, a `NotImplementedError``
             will be raised.
-        rng : `numpy.random.Generator`, optional
-            Pseudorandom number generator state. When `rng` is None, a new
-            `numpy.random.Generator` is created using entropy from the
-            operating system. Types other than `numpy.random.Generator` are
-            passed to `numpy.random.default_rng` to instantiate a `Generator`.
+        rng : `numpy.random.Generator` or `scipy.stats.QMCEngine`, optional
+            Pseudo- or quasi-random number generator state. When `rng` is None,
+            a new `numpy.random.Generator` is created using entropy from the
+            operating system. Types other than `numpy.random.Generator` and
+            `scipy.stats.QMCEngine` are passed to `numpy.random.default_rng`
+            to instantiate a ``Generator``.
+
+            If `rng` is an instance of `scipy.stats.QMCEngine` configured to use
+            scrambling and `shape` is not empty, then each slice along the zeroth
+            axis of the result is a "quasi-independent", low-discrepancy sequence;
+            that is, they are distinct sequences that can be treated as statistically
+            independent for most practical purposes. Separate calls to `sample`
+            produce new quasi-independent, low-discrepancy sequences.
```

Comment:
As discussed offline. I am not certain where to draw the line and I also think that it is going too deep for a doc in any cases. I would not discuss it and stick to saying sequences using scrambling and a different seed are independent. In many papers such as Art2020 https://arxiv.org/abs/2008.08051 (p8) this is how we do and consider this to be independent. Nevertheless, I sent an email to Sergei to have his take on that.

Reply:
Any word on this?

Reply:
Not yet. People are busy with closing the year I suppose 😄 We can keep it as it is for now.